### PR TITLE
ROC-5489 Draw order and copy doc in case documents

### PR DESCRIPTION
**Pull request by simad**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5489

### Change description ###

On draw order, we copy the draft order doc in case documents

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java [hunk 1]
```diff
@@ -24,9 +27,12 @@
     @Autowired
     public CallbackHandlerFactory(
         MoreTimeRequestedCallbackHandler moreTimeRequestedCallbackHandler,
-        Optional<GenerateOrderCallbackHandler> generateOrderCallbackHandler) {
+        Optional<GenerateOrderCallbackHandler> generateOrderCallbackHandler,
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java]: We should avoid passing Optional in parameters.

Comment (simad) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java]: 👍 will address as part of the base PR

Comment (simad) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java]: Done! changed it in https://github.com/hmcts/cmc-claim-store/pull/893. Please ignore this in this PR :D

Comment (simad) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java]: 🤹‍♀

**Pull request merged**